gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -331,6 +339,82 @@ public void reupload_success_singleWrite_singleUploadChunk() throws Exception {
     }
   }
 
+  @Test
+  public void upload_success_ioException_singleWrite_singleUploadChunk() throws Exception {
+    byte[] testData = new byte[MediaHttpUploader.MINIMUM_CHUNK_SIZE];
+    new Random().nextBytes(testData);
+    int uploadChunkSize = testData.length * 2;
+
+    MockHttpTransport transport =
+        mockTransport(
+            emptyResponse(HttpStatusCodes.STATUS_CODE_NOT_FOUND),
+            resumableUploadResponse(BUCKET_NAME, OBJECT_NAME),
+            new IOException("upload IOException"),
+            // "308 Resume Incomplete" - failed to upload anything (no "Range" header)
+            emptyResponse(308),
```

Comment:
Can we refactor repeated `308` in the file into a `private static final` field ?

Reply:
Done.